src/pages/Transaction/utils.tsx

```diff
@@ -235,8 +235,8 @@ export function useTransactionBalanceChanges(txn_version: string) {
             ? BigInt(-a.amount)
             : BigInt(a.amount),
         asset: {
-          decimals: a.metadata.decimals,
-          symbol: a.metadata.symbol,
+          decimals: a.metadata?.decimals,
```

Comment:
this causes the page not to crash anymore?

Reply:
Yes, the `metadata` object was null, and there were no error boundaries, so the entire page crashed.